src/assets/javascript/script.js

```diff
@@ -1,23 +1,75 @@
 //Navbar toggle
 const navToggle = () => {
-    const burger = document.querySelector('.burger');
-    const navList = document.querySelector('.nav-list');
-    const navLinks = document.querySelectorAll('.nav-list li');
-
-    burger.addEventListener('click', () => {
-        // Toggle Navbar
-        navList.classList.toggle('active');
-        burger.classList.toggle('toggle');
-
-        // Animate Links
-        navLinks.forEach((link, index) => {
-            if(link.style.animation) {
-                link.style.animation = '';
-            } else {
-                link.style.animation = `navLinkFade 0.5s ease forwards ${index / 7 + 0.2}s`;
-            }
-        });
+  const burger = document.querySelector('.burger');
+  const navList = document.querySelector('.nav-list');
+  const navLinks = document.querySelectorAll('.nav-list li');
+
+  burger.addEventListener('click', () => {
+    // Toggle Navbar
+    navList.classList.toggle('active');
+    burger.classList.toggle('toggle');
+
+    // Animate Links
+    navLinks.forEach((link, index) => {
+      if (link.style.animation) {
+        link.style.animation = '';
+      } else {
+        link.style.animation = `navLinkFade 0.5s ease forwards ${
+          index / 7 + 0.2
+        }s`;
+      }
+    });
+  });
+};
+
+navToggle();
+
+// GitHub Projects
+// ##############################################
+
+var githubprojectsdomelement = document.getElementById('githubprojects');
+
+let githubprojects = [
+  {
+    name: 'chryz-hub/chryz-hub.github.io',
+  },
+  {
+    name: 'chryz-hub/chryz-hub.github.io',
+  },
+  // Here can be added some more projets if needed
+];
+
+githubprojects.forEach((project) => {
+  getproject(project.name);
```

Comment:
maybe we can discuss the method naming convention here due getproject could be a little bit missleading

Reply:
Lol, I merged at the same time 🤣

Reply:
Yeah, we can work on the naming convention